### ISSUE TITLE
Maintain S&P Datasets

### ISSUE THREAD

**Issue by rgrp** [datasets/registry#66]
We need a maintainer for these 2 S&P datasets. Maintainer means someone to run the scripts regularly:

* https://github.com/datasets/s-and-p-500-companies [Update daily or weekly]
* https://github.com/datasets/s-and-p-500 - historical S&P data [monthly] (may want renaming!)

**Comment by EnricGTorrents** [comment]
I could also maintain these two, no problem.

**Comment by rgrp** [comment]
@EnricGTorrents that would be great. Do you want to fork them now and try and do the regular update and then do a pull request. Especially on the S&P Companies you may need to sort some stuff by hand in my experience. Once we have done one we can merge it in.

**Comment by EnricGTorrents** [comment]
Yes, it's better to merge once I am familiar with the process. Just sent an update on the Investor Flow of Funds package, later on today I will send you a pull request on these two.

**Comment by rgrp** [comment]
@EnricGTorrents how is this going?

**Comment by rgrp** [comment]
@EnricGTorrents how are you doing here?

**Comment by rgrp** [comment]
@EnricGTorrents hey there - is this something you can take on - if not that's no problem but then we would look for someone else to act as maintainer for the time being :-)

**Comment by abiola-adeoye** [comment]
Good day, in the spirit of contributing to open source I'd like to take up the task of maintaining the two datasets.
@rufuspollock @anuveyatsu

**Comment by rufuspollock** [comment]
@abiola-adeoye great. Could you jump on discord see link on datahub.io and we coordinate there.

**Comment by abiola-adeoye** [comment]
> [@abiola-adeoye](https://github.com/abiola-adeoye) great. Could you jump on discord see link on datahub.io and we coordinate there.

Yes I can, would join soon

**Comment by rufuspollock** [comment]
I think at least https://github.com/datasets/s-and-p-500-companies is now getting regularly updated.

I think https://github.com/datasets/s-and-p-500 should be renamed to s-and-p-500-shiller or similar - but that needs a bit of thought.

And would be nice to get a S&P 500 financials dataset.
